webhook_to_fedora_messaging/main.py

```diff
@@ -35,13 +31,11 @@ def create_app():
 
     # Then load the variables up from the custom configuration file
     try:
-       confdata = get_config()
+        confdata = get_config()
+        main.config.from_mapping(confdata)
+        db.init_app(main)
+        dictConfig(confdata["logsconf"])
     except ConfigError as expt:
         logging.error(f"Exiting - Reason - {str(expt)}")
```

Comment:
It says "exiting" but it's neither exiting nor raising the exception. I think you should add a `raise` here so that the exception is propagated upwards the call stack.